Helios/Helios.cpp

```diff
@@ -327,6 +328,11 @@ void Helios::handle_state_modes()
   uint8_t magnitude = (uint8_t)(holdDur / MENU_HOLD_TIME);
   // whether the user has held the button longer than a short click
   bool heldPast = (holdDur > SHORT_CLICK_THRESHOLD);
+
+  // flash red briefly when locked and short clicked
+  if (has_flag(FLAG_LOCKED) && holdDur < SHORT_CLICK_THRESHOLD) {

```

Comment:
You calculate hold During > short click threshold just above. HeldPast, just use that? Compiler would optimize it to reuse that if you used a <= here but since you use a < and heldPast is calculated with > it can't just reuse it in an optimization.